ui/apps/platform/src/Containers/Vulnerabilities/WorkloadCves/Overview/WorkloadCvesOverviewPage.tsx

```diff
@@ -176,20 +172,11 @@ function WorkloadCvesOverviewPage() {
     } = useWorkloadCveViewContext();
     const currentVulnerabilityState = useVulnerabilityState();
 
-    const [observedCveMode, setObservedCveMode] = useURLStringUnion(
-        'observedCveMode',
-        observedCveModeValues
-    );
-
-    // TODO Once the 'ROX_PLATFORM_CVE_SPLIT' flag is removed, we can get rid
-    // of the `observedCveMode` state and potentially abstract the detection of "zero cve view"
+    // TODO We can potentially abstract the detection of "zero cve view"
```

Comment:
➕  I agree this isn't ideal. It still seems a bit awkward to have a "zero cve view" in the view dedicated to CVEs, but I suppose that is the tradeoff with not having a more cohesive inventory view at this point.